File: Problem Solving/String/DuplicateCharacters/DuplicateChar#1.py
def duplicate(str):
    countChar={}
    for ch in str:
        if ch in countChar:
            countChar[ch]=countChar[ch]+1
        else:
            countChar[ch]=1

    for ch,count in countChar.items():
        if count>1:
            print(ch,count)
# def duplicate(str):
#     for ch in set(str):
#         if str.count(ch)>1:
#             print(ch,str.count(ch))

duplicate("Programming")

File: Problem Solving/String/DuplicateCharacters/DuplicateChar#1.js
function duplicate(str){
    let countMap=new Map();
    for(let ch of str){
        if(countMap.has(ch)){
            countMap.set(ch,countMap.get(ch)+1)
        }else{
            countMap.set(ch,1)
        }
    }
    countMap.forEach((value,key)=>{
        if(value>1){
            console.log(key)
        }
    })
}


duplicate("Programming")